ansible/roles/dev-desktop/tasks/dependencies.yml

```
@@ -63,6 +63,11 @@
       - capnproto
       - libcapnp-dev
       - zsh
+      # Needed to build libgccjit(for running `cg_gcc` on architectures other than x86)
```

```suggestion
      # The following 4 packages are needed to build libgccjit(for running `cg_gcc` on architectures other than x86)
```
We need a way to tell that all the following packages are needed for this purpose. otherwise it's hard to tell what packages we refer to